dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -176,6 +176,9 @@ http2.resetFramesWindow.desc= Specifies the window of time during which reset fr
 http2.maxStreamsRefused=Maximum streams refused
 http2.maxStreamsRefused.desc=Specifies the maximum number of streams that are refused after the maximum concurrent streams limit is reached. If this value is exceeded, the connection is closed. Set this parameter to 0 to disable the check.
 
+http2.maxHeaderBlockSize=Maximum header block size
+http2.maxHeaderBlockSize.desc=Specifies the maximum size in bytes for the header block of individual HTTP2 streams. If this value is exceeded, the connection is closed with an error. Set this parameter to 0 to disable the check.
```

Comment:
LGTM- I think would more specifically be an "attribute" rather than a "parameter" - though I see the previous definition uses parameter and both terms are in the file. It's better to use one term consistently, particularly for translation. But that change might be out of scope for this PR. Just making a note